caproto/_commands.py

```diff
@@ -563,10 +563,11 @@ class SearchRequest(Message):
 
     def __init__(self, name, cid, version):
         size, payload = padded_string_payload(name)
-        _len = len(name)
-        if _len > 40:
-            raise CaprotoValueError("EPICS imposes a 40-character limit on "
-                                    "strings. The " "string {!r} is {} "
+        rec, _, field = name.partition('.')
+        _len = len(rec)
+        if _len > 60:
+            raise CaprotoValueError("EPICS 3.14 imposes a 60-character limit "
```

Comment:
Have you encountered any documentation about how different versions of EPICS handle this? The version hardcoded here (3.14) may not match the actual `version` that we know here.

Reply:
See the link in the commit message http://www.aps.anl.gov/epics/tech-talk/2005/msg00083.php

Reply:
I think we can safely ignore anything below 3.14.

Reply:
Fair enough. I like the idea that if there are any Reagan-era Star Wars satellites out there, we could talk to them with Python 3.6. But until we find one to test against.... ;-)